core/domain/takeout_service_test.py

```diff
@@ -33,9 +33,10 @@
 
 (
     base_models, collection_models, email_models,
-    exploration_models, feedback_models, skill_models, topic_models,
-    suggestion_models, user_models, story_models, question_models,
-    config_models) = models.Registry.import_models([
+    exploration_models, feedback_models, skill_models,
+    topic_models, suggestion_models, user_models,
+    story_models, question_models, config_models
+    ) = models.Registry.import_models([
```

Comment:
Deindent by 4 to match opening paren indentation.

Reply:
Done